starwars-characters/src/character.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Link} from 'react-router-dom';
+
+class Character extends React.Component{
+    constructor(props){
+        super(props);
+
+        this.state = {
+          character: [],
+          isLoading: true,
+          Url: null
+        }
+    }
+   
+      componentDidMount(){
+        fetch() 
```

Comment:
If you made all the other changes you should be able to access whatever id was passed in the url with : 
`this.props.match/params.id`
so it would look like: 
`fetch(`https://swapi.co/api/people/${this.props.match.params.id}`)`
(or whatever the propper api call is)